caravel/assets/javascripts/SqlLab/components/DatabaseSelect.jsx

```diff
@@ -11,7 +14,7 @@ class DatabaseSelect extends React.Component {
       databaseId: null,
     };
   }
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
componentDidMount is usually used for code that requires that the dom has been rendered, while `componentWillMount` gets called right before the component is rendered. in this case, you're right, we can use either.